packages/pds/tests/sync/subscribe-repos.test.ts

```diff
@@ -273,6 +289,41 @@ describe('repo subscribe repos', () => {
     verifyHandleEvent(handleEvts[1], bob, 'bob2.test')
   })
 
+  it('syncs tombstones', async () => {
+    const baddie1 = (await sc.createAccount("baddie1.test", {
+      email: 'baddie1@test.com',
+      handle: 'baddie1.test',
+      password: 'baddie1-pass',
+      displayName: undefined,
+      description: undefined,
+    })).did
+    const baddie2 = (await sc.createAccount("baddie2.test", {
+      email: 'baddie2@test.com',
+      handle: 'baddie2.test',
+      password: 'baddie2-pass',
+      displayName: undefined,
+      description: undefined,
+    })).did
+
+    for (var did of [baddie1, baddie2]) {
```

Comment:
Should this be a `const` instead of a `var`? Looks like you used a `const` in the other test so this might have been a typo.